addons/jest/src/components/Panel.tsx

```diff
@@ -58,34 +66,60 @@ const SuiteTotals = styled(({ successNumber, failedNumber, result, className, wi
   },
 }));
 
-const SuiteProgress = styled(({ successNumber, result, className }) => (
-  <div className={className} role="progressbar">
-    <span style={{ width: `${(successNumber / result.assertionResults.length) * 100}%` }} />
-  </div>
-))(({ theme }) => ({
-  width: '30px',
-  backgroundColor: theme.color.negative,
-  height: '6px',
-  top: '3px',
-  position: 'absolute',
-  left: 0,
-  overflow: 'hidden',
-  appearance: 'none',
-
-  '& > span': {
-    backgroundColor: theme.color.positive,
-    bottom: 0,
-    position: 'absolute',
-    left: 0,
-    top: 0,
-  },
-}));
+const SuiteProgressPortion = styled.div<{ color: any; progressPercent: number }>(
+  ({ theme, color, progressPercent }) => ({
+    height: '6px',
+    top: '3px',
+    width: `${progressPercent}%`,
+    backgroundColor: color,
+  })
+);
 
 interface ContentProps {
   tests: Test[];
   className?: string;
 }
 
+const getTestsByTypeMap = (result: any) => {
+  const testsByType: Map<string, any> = new Map();
+  result.assertionResults.forEach((assertion: any) => {
+    // using switch to allow for new types to be added
+    switch (assertion.status) {
```

Comment:
why not just use `assertion.status` instead of switch because the logic is the same in each branch?

Reply:
Oh my goodness, nice find. I could definitely use assertion.status instead. No need for the switch at all

Reply:
Done!